radish34/bin/setup.sh

```diff
@@ -9,7 +9,7 @@ printf "\n${GREEN}*** Starting zokrates container ***${NC}\n"
 docker-compose up -d zkp
 
 # delay needed to ensure all container are in running state.
-sleep 5
+sleep 10
```

Comment:
🤦‍♂️I could use more sleep... computers should not.

Reply:
> 🤦‍♂️I could use more sleep... computers should not.

lol. +1

Reply:
Every 5 seconds of sleep is valuable! I think this should be replaced by polling a healthcheck on the container or something similar. There is an open issue for adding container healthchecks: https://github.com/ethereum-oasis/baseline/issues/51

Reply:
Yeah, I'm looking at the healthcheck issue today.